sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/samples/README.md

```diff
@@ -10,8 +10,8 @@ urlFragment: schemaregistry-avro-samples
 
 # Azure Schema Registry serializer library samples for Java
 
-Azure Schema Registry Avro Serializer samples are a set of self-contained Java programs that demonstrate serialization 
-and deserialization of data for which the schema is stored in Schema Registry service. Each sample focuses on a 
+Azure Schema Registry Avro Serializer samples are a set of self-contained Java programs that demonstrate serialization
```

Comment:
Cannot comment on the file above but the modulename should be updated too in the module-info.java file above.